### PR TITLE
added md5 hash

### PR DESCRIPTION
**Pull request by Jo3yVX**

### CODE REVIEW

File: consts.js [hunk 1]
```diff
@@ -106,7 +106,12 @@ var thirteenStrings = [
     "wkluwhhq",
     "Wkluwhhq",
     "WKLUWHHQ",
-
+   
+    //md5 hash
+    "c51ce410c124a10e0db5e4b97fc2af39"
```
Comment (Almenon) [inline consts.js:111]:
```suggestion
    "c51ce410c124a10e0db5e4b97fc2af39",
```

File: consts.js [hunk 2]
```diff
@@ -106,7 +106,12 @@ var thirteenStrings = [
...
+    //also md5 hash but UPPERCASE
+    "C51CE410C124A10E0DB5E4B97FC2AF39"
```
Comment (Almenon) [inline consts.js:113]:
```suggestion
    "C51CE410C124A10E0DB5E4B97FC2AF39",
```

File: consts.js [hunk 3]
```diff
@@ -106,7 +106,12 @@ var thirteenStrings = [
     "wkluwhhq",
     "Wkluwhhq",
     "WKLUWHHQ",
-
+   
+    //md5 hash
+    "c51ce410c124a10e0db5e4b97fc2af39"
+    //also md5 hash but UPPERCASE
```
Comment (Almenon) [inline consts.js:112]: you remembered to account for uppercase?? :O 

You'll go far as a programmer :)

**Review by Almenon — CHANGES_REQUESTED** [review]

**Comment by Almenon** [comment]
Not a merge blocker but note that md5 is insecure and can be hacked by the enemy (-13) at any time. For a secure hash please use SHA-512.

File: consts.js (hunk 3 above)
Comment (Jo3yVX) [reply consts.js:112]: I know that since the very first word I said is "thirteen" but not "mama".

**Comment by Jo3yVX** [comment]
Neglecting MD5 (but not SHA-512) just because it's insecure would be like eliminating disabled people because they're incomplete. Both have the right to life, and I will be really disappointed if MD5 Hash is not allowed to be in such a supercalifragilisticexpialidocious project like "is-13".

> Not a merge blocker but note that md5 is insecure and can be hacked by the enemy (-13) at any time. For a secure hash please use SHA-512.

**Comment by Jo3yVX** [comment]
Neglecting MD5 Hash just because it's insecure would be like eliminating disabled people because they are incomplete. Both have the right to life, and people will be really disappointed if MD5 Hash is not allowed to be in such a supercalifragilisticexpialidocious project like "is-13".

> Not a merge blocker but note that md5 is insecure and can be hacked by the enemy (-13) at any time. For a secure hash please use SHA-512.

**Review by Almenon — APPROVED** [review]

**Pull request merged**